frontend/lib/src/dataframes/arrowTypeUtils.ts

```diff
@@ -35,22 +35,13 @@ export type DataType =
   | bigint // period
 
 export enum PandasIndexTypeName {
-  CategoricalIndex = "categorical",
-  DatetimeIndex = "datetime",
-  Float64Index = "float64",
-  Int64Index = "int64",
   RangeIndex = "range",
```

Comment:
If this is the only element left in the enum, maybe make it a constant? Or do we expect more elements to be added again?

Reply:
Yep, changed it to constant